COPYRIGHT-NOTICES.md

```diff
@@ -0,0 +1,138 @@
+<!-- SPDX-License-Identifier: CC-BY-4.0 -->
+
+# Copyrights and OpenJS Foundation projects
+
+## Ownership of Copyrights in OpenJS Foundation Contributions
+
+When source code, documentation and other content is contributed to an Open JS Foundation
+project, the copyrights in those contributions remain owned by the original
+copyright holders.
+
+The copyrights are not _assigned_ to OpenJS Foundation. Instead, they are _licensed_ for
+distribution as part of the project. Whether a project uses the DCO or a CLA,
```

Comment:
Could we explain these acronyms?

```html
Whether a project uses a
<abbr title="Developer Certificate of Origin">[DCO][]</abbr>
or a
<abbr title="Contributor License Agreement">CLA</abbr>

[DCO]: https://developercertificate.org/
```

Reply:
@brianwarner 

I think I mentioned this before, but this hasn't always been the case.

The JS Foundation had the original author of Mocha and AFAIK _every contributor_ to reassign the copyright to the JS Foundation.

I'm going to assume that means we need to change the copyright notice (in `LICENSE` file) from:

```
Copyright (c) 2011-2018 JS Foundation and contributors, https://js.foundation
```

to

```
Copyright (c) 2011-2020 OpenJS Foundation and contributors, https://openjsf.org
```

???

If this is the case, please add guidance for those projects that the Foundation _does_ retain the copyright to.

Reply:
Hey @boneskull,

Yes, correct, my understanding is that the foundation's stance on copyright has gone through a number of iterations. The current one is more permissive than the prior two iterations, where now adoption of a CLA is optional, with the decision to be made by the project's maintainer(s).

Because the JS Foundation merged into the OpenJS Foundation, it is factually accurate to update "JS Foundation" to "OpenJS Foundation."  This came up [in another comment thread](https://github.com/openjs-foundation/cross-project-council/pull/414#discussion_r353826217), and we can add this to the document when we revise it.

Reply:
> Could we explain these acronyms?
> 
> ```
> Whether a project uses a
> <abbr title="Developer Certificate of Origin">[DCO][]</abbr>
> or a
> <abbr title="Contributor License Agreement">CLA</abbr>
> 
> [DCO]: https://developercertificate.org/
> ```

I just pushed a patch which expands these acronyms and links to the upstream documents.

Reply:
> Hey @boneskull,
> 
> Yes, correct, my understanding is that the foundation's stance on copyright has gone through a number of iterations. The current one is more permissive than the prior two iterations, where now adoption of a CLA is optional, with the decision to be made by the project's maintainer(s).
> 
> Because the JS Foundation merged into the OpenJS Foundation, it is factually accurate to update "JS Foundation" to "OpenJS Foundation." This came up [in another comment thread](https://github.com/openjs-foundation/cross-project-council/pull/414#discussion_r353826217), and we can add this to the document when we revise it.

I just pushed a patch which explicitly adds this into the guidance.